code/controllers/master.dm

```diff
@@ -124,7 +141,8 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	var/delay = 50 * ++Master.restart_count
```

Comment:
All of this proc needs to have `Master.` replaced with `Master?.` or it will just runtime

Reply:
Isn't restart_count, restart_timeout and restart_clear static, so they will always exist?
That's why I only added the check if Master exists for processing

Reply:
actually im pretty sure this works now because Master is a typed variable and these access static variables. amazing.

Reply:
what the fuck

Reply:
Well in my testing this way of accessing the vars always worked, so I guess its fine the way it currently is